study.md

```diff
@@ -21,7 +21,11 @@ In your own words, explain one reason why we use "this". In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+<!-- We use this inside an object so we don't have to keep calling the object's name.
+
```

Comment:
That's definitely one reason, another is that we might use "this" is to avoid ambiguity by making sure we are
correctly referring to the current object, instead of perhaps another globally
defined variable with the same name as the object.